dxp/build_process.md

```diff
@@ -72,6 +72,65 @@ The `liferay-npm-bridge-generator` is only executed when there is a `.npmbridger
 
 > To learn more about how to use this tool read ["How to use liferay npm bridge generator" in the wike of the liferay-js-toolkit repository](https://github.com/liferay/liferay-js-toolkit/wiki/How-to-use-liferay-npm-bridge-generator).
 
+### Build SCSS
+
+There are two ways that DXP handles building `.scss` files:
+
+-   [CSS Builder Gradle Plugin](https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-css-builder/README.markdown)
+-   Liferay npm bundler with [sass-loader](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler-loader-sass-loader) and [css-loader](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler-loader-css-loader) via JavaScript
```

Comment:
I think you should drop the reference to the sass-loader, because we don't use it, and AFAIK, don't want to use it:

```
$ yarn why liferay-npm-bundler-loader-sass-loader
yarn why v1.22.5
[1/4] 🤔  Why do we have the module "liferay-npm-bundler-loader-sass-loader"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
error We couldn't find a match!
```

Reply:
> don't want to use it

:cry: That hurt...

Reply:
![tough-love](https://user-images.githubusercontent.com/7074/93446913-10638680-f8d2-11ea-9618-4a75ad042d21.gif)

Reply:
😱 oh sorry, what a pity! I'm sorry @izaera haha